### PR TITLE
Builds images for releases

### PR DESCRIPTION
**Pull request by robotrapta**
Adding a github action which builds OS images, and saves them as release artifacts as appropriate.

Also renamed things in the repo to be less MNS-specific, because this repo will build & release various flavors with and without MNS.

### CODE REVIEW

File: .github/workflows/build-images.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          # faster test build.
+          ./build-docker.sh -c gl-config
+        fi
+
+    # Print output directory files
+    - name: List output files
+      run: ls -lh deploy
+
+    - name: Upload built image
+      if: env.IS_RELASE == '1'
```
Comment (mjvogelsong) [inline .github/workflows/build-images.yaml:50]:
```suggestion
      if: env.IS_RELEASE == '1'
```

File: .github/workflows/build-images.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      uses: actions/upload-artifact@v3
+      with:
+        name: rpios-image
+        path: ./deploy/image_*.img.xz
+        if-no-files-found: error
+
+    - name: Upload debug artifacts
+      if: env.IS_RELASE == '1'
```
Comment (mjvogelsong) [inline .github/workflows/build-images.yaml:58]:
```suggestion
      if: env.IS_RELEASE == '1'
```

File: GL-README.md [hunk 3]
```diff
@@ -1,20 +1,28 @@
...
 
-Use this repo to build the `.img` file which gets burned to an SD card.  When that SD card is booted on the Raspberry Pi, 
-it runs the [Groundlight Monitoring Notification Server](https://github.com/groundlight/monitoring-notification-server).
-The MNS provides a simple GUI to configure a Groundlight detector, grab images from it, and send notifications when
+There are several different images available, depending on your needs, from smallest to largest:
+
+- **`sdk`** - A minimal image with just the Python SDK installed.  This is the smallest image, and is suitable for running the SDK on a Raspberry Pi Zero W.  It is also suitable for running the SDK on a Raspberry Pi 3 or 4, if you don't need the GUI.
+- **`mns`** - an image with the [Groundlight Monitoring Notification Server (MNS)](https://github.com/groundlight/monitoring-notification-server) installed for headless use.  "Headless" means as server, without a GUI.  The MNS provides a simple GUI to configure a Groundlight detector, grab images from it, and send notifications when
```
Comment (mjvogelsong) [inline GL-README.md:9]: `"Headless" means as server, without a GUI. The MNS provides a simple GUI` -- I'm a little confused by that statement. Does this mean you can open an interface on a web browser pointed at the raspberry pi, but can't use this with a raspberry pi's screen?

**Review by mjvogelsong — APPROVED** [review]
Cool!

File: .github/workflows/build-images.yaml (hunk 2 above)
Comment (robotrapta) [reply .github/workflows/build-images.yaml:58]: 🙏🏾

File: GL-README.md (hunk 3 above)
Comment (robotrapta) [reply GL-README.md:9]: That's exactly what it means.  It has a server which serves the HTML, but you need to run the browser somewhere else.  I'll try to write that more clearly.

**Pull request merged**